analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/EmptyNullableValueAccess.CSharp9.cs

```diff
@@ -140,12 +149,12 @@ public void PatternMatching(Repro_6682? arg, bool condition)
 
         if (arg is { SomeProperty: true })  // A null check
         {
-            var value = arg.Value;  // FIXME Non-compliant FP
+            var value = arg.Value;          // Compliant
```

Comment:
As this explicitly fixes a reported FP, please update the description to contain `Fixes #6682`. The automation will start moving the issue around with next actions. To catch up manually:
* drag it in "In progress" column, it will put your face on it
* remove this PR from Kanban, as the issue will represent it
* remove assignee from the PR, as the issue will represent it